client/src/modules/Score/components/ScoreTable/index.tsx

```diff
@@ -15,16 +15,20 @@ import { SettingsModal } from 'modules/Score/components/SettingsModal';
 import { Store } from 'rc-field-form/lib/interface';
 import { useLocalStorage } from 'react-use';
 import { CoursesTasksApi, CourseTaskDto } from 'api';
+import useWindowDimensions from 'utils/useWindowDimensions';
 
 type Props = CoursePageProps & {
   onLoading: (value: boolean) => void;
   activeOnly: boolean;
 };
 
+type TFixedColumn = boolean | 'left';
```

Comment:
And seems a bit strange. Why not just `true` and `false`?
Or `left` and `right` for example?

Reply:
ok, noted.

the property type (FixedType) that sets column fixed is `boolean | 'left' | 'right'` - it stated that way in columnSharedType from antd lib;

to set column fixed left we can use either `true` or `left`, Githab column was set `left` before my manipulations, therefore I used `left` to set it back like it was; to unfix column we can only use `false`; I also omitted `right` on my type because on this particular case column shouldn't be right fixed. That's the logic behind making that type.

It'll work if I use only boolean values, should I change it that way?

Reply:
have removed that type completely, `fixedColumn` set `boolean`, and `githubColumn.fixed = fixedColumn ? 'left' : false;` get type inferred from column type: `(property) ColumnSharedType<RecordType>.fixed?: FixedType | undefined`